jOOQ/src/main/java/org/jooq/DSLContext.java

```diff
@@ -8362,6 +8362,62 @@
      */
     Queries ddl(Table<?> table, DDLFlag... flags);
 
+    /**
+     * Generate the complete creation script for tables.
+     *
+     * @see #ddl(Table[], DDLFlag...)
+     */
+    Queries ddl(Table[] tables);
```

Comment:
This will get a wildcard: `Table<?>` to avoid the raw type

Reply:
Also, the version of this API that doesn't specify any `DDLFlag` can profit from varargs